Samples/ios/SampleApp/SampleAppiOS/SampleMainViewController.m

```diff
@@ -136,6 +136,10 @@ - (void)loadEvents
         }
         
         _events = events;
+        _keys = [[_events allKeys] sortedArrayUsingComparator:^NSComparisonResult(id  _Nonnull obj1, id  _Nonnull obj2) {
+            return [obj1 compare:obj2];
+        }];
+        
```

Comment:
Can it be a small helper method? It's not important, but keys sorting + tableView reload gets repeated in loadEvents  before this async call.